robottelo/ui/factory.py

```diff
@@ -582,6 +582,19 @@ def edit_param(session, org=None, loc=None,  force_context=True, **kwargs):
     Settings(session.browser).update(**update_args)
 
 
+def remove_param(session, org=None, loc=None,  force_context=True, **kwargs):
```

Comment:
why do we need that method at all? It doesn't sound as factory method and is redundant in its idea completely
If you need navigation entity - do it in `remove_parameter`
If you need switching to necessary organization or location - do it in test or use session_org approach

Reply:
@oshtaier  `remove_param` will help to test the `"//div/span[contains(@class, 'editable-clear-x')]`" whether it is removing the  parameter or not and I put it in factory because it can be used in any test which is related to remove parameter/update without value (or make parameter 'Empty').